addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_nodes.py

```diff
@@ -177,12 +177,25 @@ def __gather_mesh(blender_object, export_settings):
             edge_split.split_angle = blender_object.data.auto_smooth_angle
             edge_split.use_edge_angle = not blender_object.data.has_custom_normals
 
+        armature_modifiers = {}
+        if export_settings[gltf2_blender_export_keys.SKINS]:
+            # temprorary disable Armature modifiers if exporting skins
+            for idx, modifier in enumerate(blender_object.modifiers):
+                if modifier.type == 'ARMATURE':
+                    armature_modifiers[idx] = modifier.show_viewport
+                    modifier.show_viewport = False
+
         if bpy.app.version < (2, 80, 0):
             blender_mesh = blender_object.to_mesh(bpy.context.scene, True, 'PREVIEW')
         else:
             blender_mesh = blender_object.to_mesh(bpy.context.depsgraph, True)
         skip_filter = True
 
+        if export_settings[gltf2_blender_export_keys.SKINS]:
+            # restore Armature modifiers
+            for key, val in armature_modifiers.items():
```

Comment:
nit: let's use the `idx, modifier` variable names to match the block above.

Reply:
I'll settle for `idx, show_viewport` as that is really what's in the dict